wiki/en/Tips-Tricks-More.md

```diff
@@ -80,21 +80,92 @@ Here is the script:
  jack_connect Jamulus:'output right' system:playback_2
 ~~~
 
-### Using ctrlmidich for MIDI controllers
+### Using `--ctrlmidich` for MIDI controllers
 
-The volume fader, pan control and mute and solo buttons in the Client's mixer window strips can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There is one global MIDI channel parameter (1-16) and two parameters you can set for each item controlled: `offset` and `consecutive CC numbers`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and then specify the items you want to control (f = volume fader; p = pan; m = mute; s = solo; o = mute myself) with the offset (CC number to start from) and number of consecutive CC numbers. There is one exception that does not require establishing consecutive CC numbers which is the "Mute Myself" command - it only requires a single CC number as it is only applied to one's own audio stream. Take the following example:
+When this option is used on the command line, Jamulus will display a server channel number assigned to each client, which can be used to control the channel using MIDI CC numbers.  (Note: This is only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support.  It also relies on the Jamulus UI being displayed.)
```

Comment:
Do "server" and "client" need to be capitalised here?

Reply:
Ah yes.